scripts/import_missing_qc_variants_files.py

```diff
@@ -0,0 +1,124 @@
+import os
+import csv
+import subprocess
+from tqdm import tqdm
+import time
+
+# file constant
+folder_path = '/mnt/disks/data/data-directory/variant_qc/panel_variant_qc'
+output_fname = folder_path + "/combined_sorted_output_file.tsv"
+# Output tsv Gzipped file path
+output_gzipped_tsv = output_fname + ".gz"
+skipped_file_name = 'chrX.removed.add_LCR.add_XPAR.add_failed_filter_awk.tsv'
+
+# List all files in the folder
+def get_files_list():
+    files = []
+    for filename in os.listdir(folder_path):
+        file_path = os.path.join(folder_path, filename)
+        if file_path != output_fname and not skipped_file_name in file_path:
+            files.append(file_path)
+    return files
+
+
+def main_script():
+    filenames = get_files_list()
+    filenames.sort()
+
+    # write the array of objects to a csv file
+    if os.path.exists(output_fname):
+        os.remove(output_fname)  # Overwrite the file with an empty DataFrame
+        print(f"\nFile '{output_fname}' is already exist, so it is cleared first.")
+
+    # read in the files
+    with open(output_fname,"wt",encoding="utf-8") as out_file:
+        # output file header
+        output_file_reader = csv.reader(out_file)
+        output_file_header = None
+        if not os.path.getsize(output_fname) == 0:
+            output_file_header = next(output_file_reader)
+            print(output_file_header)
```

Comment:
Some questions regarding this code:
- On line 34, you open the file in "wt" mode, meaning:
  - if file exists, it is cleared (i.e it's size will be 0 ), and opening in write and text modes.
  - On line 38, you check for the size of the output file, but since we open it in in write mode, it's going to be 0, so this code block is never entered.

So what is the intent of the code?
To add to that, on lines 29-31, if the file existed before, it would have been removed, so it's size will either be 0 if it did not exist, or it will be 0 since it existed, was thereafter removed, and then opened in write mode, which will also clear the file.